diff --git a/CMakeLists.txt b/CMakeLists.txt
index ad20777..c121d72 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1642,7 +1642,7 @@ if(NOT TARGET uninstall)
       ${CMAKE_CURRENT_BINARY_DIR}/CMake/cmake_uninstall.cmake
       IMMEDIATE @ONLY)
 
-  add_custom_target(uninstall
-      COMMAND ${CMAKE_COMMAND} -P
-      ${CMAKE_CURRENT_BINARY_DIR}/CMake/cmake_uninstall.cmake)
+#  add_custom_target(uninstall
+#      COMMAND ${CMAKE_COMMAND} -P
+#      ${CMAKE_CURRENT_BINARY_DIR}/CMake/cmake_uninstall.cmake)
 endif()
