diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6b39f0e..6593b39 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2265,11 +2265,11 @@ add_subdirectory(fuzz)
 # The EXPORT call is paired with one in liblib.
 set(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/cmake/netCDF)
 
-install(EXPORT netCDFTargets
-  DESTINATION ${ConfigPackageLocation}
-  COMPONENT headers
-  NAMESPACE netCDF::
-  )
+#install(EXPORT netCDFTargets
+#  DESTINATION ${ConfigPackageLocation}
+#  COMPONENT headers
+#  NAMESPACE netCDF::
+#  )
 
 include(CMakePackageConfigHelpers)
 CONFIGURE_PACKAGE_CONFIG_FILE(
